README.md

```diff
@@ -1 +1,12 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### This is Ram M Kripa's Homework 1 submission.
+I think I understand the **github workflow** for the most part now.
```

Comment:
Good job using some different markdown features! I would suggest adding some more detail to your commit comment when you make changes, overall great job!